packages/tsconfig/base.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
The files in `tsconfig` are based on the examples in the [official Turborepo repo](https://github.com/vercel/turbo/tree/main/examples/with-create-react-app/packages/tsconfig)